src/flask_jsonrpc/site.py

```diff
@@ -193,7 +203,7 @@ def dispatch(
             # TODO: Improve the checker to return type
             view_fun_annotations = t.get_type_hints(view_func)
             view_fun_return: t.Optional[t.Any] = view_fun_annotations.pop('return', None)
-            if resp_view is not None and view_fun_return is None:
+            if validate and resp_view is not None and view_fun_return is None:
```

Comment:
What is the purpose if this new `validate` property? To disable type checking?

Reply:
The `validate` parameter already existed on the project, it is useful in some cases where the project doesn't support some complex types, in that way, the developer can use these unsupported types and open an issue here to fix that, in the other words, to not block the development from external developers.

I added it here because in my tests I found a bug here and I solved it with added this parameter here, ;).

Another bug that I found was https://github.com/cenobites/flask-jsonrpc/issues/387

Reply:
See here: https://github.com/cenobites/flask-jsonrpc/blob/master/src/flask_jsonrpc/wrappers.py#L76